requirements-test.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
@tkdchen Do you know, should it be run with 3.9?

Reply:
I'd say it doesn't matter